addons/im_livechat/static/src/core/common/thread_actions.js

```diff
@@ -0,0 +1,17 @@
+import { patch } from "@web/core/utils/patch";
+
+import { threadActionsInternal } from "@mail/core/common/thread_actions";
+
+patch(threadActionsInternal, {
+    condition(component, id, action) {
+        const visitorActions = ["settings", "search-messages", "attachments"];
+        if (
```

Comment:
If I'm not mistaken, basically the same settings as in `@im_livechat/embed/disabled_features.js`.

Not sure it's worth enabling search messages and attachment panel, considering live chat are generally short-lived conversations so attachment panels is mostly likely empty or barely useful.